test/queue-test.js

```diff
@@ -109,6 +112,16 @@ describe('Queue', (it) => {
     });
 
     function makeQueue(...args) {
+      if (redisUrl) {
+        if (args.length === 0) {
+          args.push({});
+        }
+        if (!args[0].redis) {
```

Comment:
Will this not `throw` when called with no `args` and a configured `BEE_QUEUE_TEST_REDIS`?

Reply:
I don't think so -- am I missing something?  I'll dig into it tomorrow....

Reply:
I don't get an error.
I added the ` it('reports a succeeded job', async (t) => { ... ` test from `queue-test.js` to `delay-test.js`.  It uses `makeQueue()` with no args, and it runs without error in the configured `BEE_QUEUE_TEST_REDIS` environment I'm using with Docker (and for which I'm about to post a PR).

Reply:
Oh I'm really sorry - I don't know what was happening yesterday but I musta had a bug in my braincompiler. This is right!